src/hooks/useUsers/useUsers.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+
+export const useUsers = ({ tenant = '' }) => {
+  const [namespaceKey] = useNamespace({ key: 'relatedUsers' });
+  const ky = useOkapiKy({ tenant });
+  const [users, setUsers] = useState([]);
+
+  const { data, isFetching, isSuccess } = useQuery({
+      queryKey: [namespaceKey, tenant],
+      queryFn: () => ky.get('users').json(),
```

Comment:
Won't ignoring the `limit` query parameter limit the results to 10 records?

Reply:
fixed